nginx-fem-staging-redirects.conf

```diff
@@ -31,15 +31,24 @@ location ~* ^/about/(?:team|publications)/?$ {
 }
 
 # FEM Projects app routes for project index page (optional trailing slash)
-location ~* ^/projects/(?:[\w-]*?/)?[\w-]*?/[\w-]+?/?$ {
+location ~* ^/projects/[\w-]*?/[\w-]+?/?$ {
     resolver 1.1.1.1;
     proxy_pass $fe_project_uri;
     proxy_set_header Host $fe_project_host;
 
     include /etc/nginx/proxy-security-headers.conf;
 }
 
-# FEM projects app: home,about and classify pages
+# FEM Projects app routes for project index page with locale (optional trailing slash)
+location ~* "^/projects/[[:alpha:]]{2,4}(?:-[[:alnum:]]{0,4})?/[\w-]*?/[\w-]+?/?$" {
```

Comment:
I think that fails because RegExr doesn’t support Posix codes. The proxy seems to work, though, since you can navigate to that URL and NginX passes the request through to the NextJS app server.